InvenTree/plugin/apps.py

```diff
@@ -30,6 +30,15 @@ def ready(self):
 
                 if not registry.is_loading:
                     # this is the first startup
+                    try:
+                        from common.models import InvenTreeSetting
+                        if InvenTreeSetting.get_setting('PLUGIN_ON_STARTUP'):
```

Comment:
Please change this to:

```python
if InvenTreeSetting.get_setting('PLUGIN_ON_STARTUP', create=False)
```

to prevent database writes during app launch